cmake/compilers/X3D_flags_intel.cmake

```diff
@@ -0,0 +1,11 @@
+# Compilers Flags for Intel
+
+set(X3D_FFLAGS "-fpp -std08 -xHost -heaparrays -safe-cray-ptr -g -traceback")
+set(X3D_FFLAGS_RELEASE "-O3 -ipo")
+set(X3D_FFLAGS_DEBUG   "-g -O0 -debug extended -traceback -DDEBUG")
+set(X3D_FFLAGS_DEV     "${X3D_FFLAGS_DEBUG} -warn all,noexternal")
+#set(CMAKE_Fortran_FLAGS "-cpp xSSE4.2 -axAVX,CORE-AVX-I,CORE-AVX2 -ipo -fp-model fast=2 -mcmodel=large -safe-cray-ptr")
+# 
+set(MKL_INTERFACE "lp64")
+set(MKL_THREADING "sequential")
+find_package(MKL CONFIG REQUIRED)
```

Comment:
I do not have the intel compiler available to test. I think the compiler flags should match the ones currently used in the Makefile. When 2decomp is an external library, MKL-related stuff are no longer needed ?

Reply:
The MKL is required at linking. I removed it but afterwards the linking did not like, for this reason i add it back. No idea why with FFTW is not a problem.

Reply:
I think MKL is needed if using intel compiler + MKL FFT. It should not be needed if using intel compiler + FFTW for instance

Reply:
To keep things simple, we can consider this is a corner case. As long as intel compiler + MKL FFT works this is ok.